### ISSUE TITLE
support needed for r6g instance types for elasticache

### ISSUE THREAD

**Issue by jpatallah** [terraform-linters/tflint#953]
## Problem
AWS added r6g instance type support in elasticache 2020-10-08: https://aws.amazon.com/about-aws/whats-new/2020/10/amazon-elasticache-now-supports-m6g-and-r6g-graviton2-based-instances/

Currently the latest version of tflint ( 0.20.2) complains that this instance type is invalid.
```hcl
1 issue(s) found:

Error: "cache.r6g.xlarge" is invalid node type. (aws_elasticache_cluster_invalid_type)

  on elasticache.tf line 6:
   6:   node_type            = var.instance_type

Reference: https://github.com/terraform-linters/tflint/blob/v0.20.2/docs/rules/aws_elasticache_cluster_invalid_type.md```
```

## Version
TFLint version 0.20.2
Terraform v0.13.3

## Request
Please add support for cache.r6g.xlarge instance types for Elasticache memcached

## Edit
created PR for the new instances types: https://github.com/terraform-linters/tflint/pull/954

**Comment by acdha** [comment]
The same update is needed for the `db` instance types: https://aws.amazon.com/about-aws/whats-new/2020/10/achieve-up-to-52-percent-better-price-performance-with-amazon-rds-using-new-graviton2-instances/
